app/src/main/scala/org/alephium/explorer/persistence/queries/HashrateQueries.scala

```diff
@@ -76,58 +86,70 @@ trait HashrateQueries extends CustomTypes {
    * round to above 10 minutes: ceiling(26.09 / 10) * 10 = 30
    * result: 8:30
    */
-  private def compute10MinutesHashrateRawString(from: TimeStamp): String = {
+
+  private val timestampTZ = "to_timestamp(timestamp/1000.0) AT TIME ZONE 'UTC'"
```

Comment:
`to_timestamp` converts seconds to date?

Reply:
yes, but seconds can have a float value to get the ms, that's why I divide by `1000.0` and not `1000`:
```
postgres=# select to_timestamp( 1646056461886/1000);
      to_timestamp      
------------------------
 2022-02-28 13:54:21+00
(1 row)

postgres=# select to_timestamp( 1646056461886/1000.0);
        to_timestamp        
----------------------------
 2022-02-28 13:54:21.886+00
(1 row)
```

Reply:
Since second is precise enough for all of your queries, `timestamp / 1000` is much more efficient

Reply:
let's tackle this with https://github.com/alephium/explorer-backend/issues/154 in another PR as it breaks most of the tests